.github/workflows/build.yml

```diff
@@ -87,8 +87,6 @@ jobs:
 
       - name: Run rustdoc lints
         uses: actions-rs/cargo@v1
-        env:
-          RUSTDOCFLAGS: '-D missing_docs -D rustdoc::missing_doc_code_examples'
```

Comment:
Thanks for this, I'll add docs eventually 👍🏻  😂

Reply:
It's Okay for now, only doc for macro were blocking